Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -39,6 +40,14 @@ public class BacktestingBrokerage : Brokerage
         private readonly object _needsScanLock = new object();
         private readonly HashSet<Symbol> _pendingOptionAssignments = new HashSet<Symbol>();
 
+        private readonly Dictionary<TimeInForce, ITimeInForceHandler> _timeInForceHandlers = new Dictionary<TimeInForce, ITimeInForceHandler>
+        {
+            { TimeInForce.GoodTilCanceled, new GoodTilCanceledTimeInForceHandler() },
+            { TimeInForce.Day, new DayTimeInForceHandler() },
+            // Custom time in force will be renamed to GTD soon and will have its own new handler
+            { TimeInForce.Custom, new GoodTilCanceledTimeInForceHandler() }
```

Comment:
This looks nice and clean! With the upcoming refactor this is going to change (again) such that the implementation will ride along w/ the order object -- but for now it's nice a clear what the mappings are.